moonlight/conversions/musicxml.py

```diff
@@ -183,6 +183,19 @@ def _glyph_to_note(glyph):
   etree.SubElement(
       pitch, 'alter').text = str(ACCIDENTAL_TO_ALTER[pitch_match.group(2)])
   etree.SubElement(pitch, 'octave').text = pitch_match.group(3)
+
+  if glyph.HasField('stem'):
+    # Check for stem direction
+    # If more of the stem is above the y_position, stem direction = up
+    # Otherwise, stem direction = down
+    # If the distances above and below are equal, set stem direction = up
+    stem_distance_above = glyph.y_position - glyph.stem.start.y
```

Comment:
This is a little off because the y position is a relative index on the staff, and other y coordinates are absolute. We have a utility for converting y positions to coordinates. Could you try:

```
from moonlight.glyphs import geometry
...
glyph_y = geometry.glyph_y(staff, glyph)
```

You'll also have to pass in the staff to this function, and add "//moonlight/glyphs:geometry" to "musicxml" in the BUILD file.